src/client/load.js

```diff
@@ -93,10 +70,46 @@ export function applyCallbacks (dataAttributeName, matchElement) {
     }
 
     for (const element of elements) {
-      element.addEventListener('load', () => {
-        element.removeAttribute(`data-${dataAttributeName}`)
+      /* __vm_cb: whether the load callback has been called
+       * __vm_l: set by onload attribute, whether the element was loaded
+       * __vm_ev: whether the event listener was added or not
+       */
+      if (element.__vm_cb) {
```

Comment:
I've could have made this an object like `__vm = {}` instead but as `__vm_l` needs to be set from the onload attribute and using an object there is too complicated,  this approach of separate variables seemed ok to me.